encoder/src/main/java/openlr/encoder/properties/OpenLREncoderProperties.java

```diff
@@ -81,6 +81,10 @@
     /** The comp time4 cache. */
     private final int compTime4Cache;
 
+    private boolean insertLrpAtAlternatePath;
```

Comment:
please add short java docs explanation for both new properties

Reply:
00c70cf1bd4f3e32f47bccb0523ebe37cbbce6e8